core/src/main/java/hudson/model/Executor.java

```diff
@@ -594,6 +653,32 @@ protected Object call(Object o, Method m, Object[] args) throws Throwable {
     }
 
     /**
+     * Finds the executor running a given process.
```

Comment:
I do not think so. As of 1.536, an `Executor` is thrown out after each use. Thus at worst it will return null for something that _was_ running earlier. But this deserves clarification.